frontend/src/pages/About/about.module.scss

```diff
@@ -97,14 +97,17 @@
 .row1 > * {
   flex: 49% 49%;
 }
-
+h1 {
+  text-align: center;
```

Comment:
same comment as below, we don't want it centred aligned. Please make it same as it was before